TeamViewerPS/Public/Remove-TeamViewerUserGroupMember.ps1

```diff
@@ -0,0 +1,101 @@
+function Remove-TeamViewerUserGroupMember {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    param(
+        [Parameter(Mandatory = $true)]
+        [securestring]
+        $ApiToken,
+
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserGroupId } )]
+        [Alias("UserGroupId")]
+        [Alias("Id")]
+        [object]
+        $UserGroup,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByUserGroupMemberId")]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserGroupMemberMemberId } )]
+        [Alias("UserGroupMemberId")]
+        [Alias("MemberId")]
+        [object]
+        $UserGroupMember,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByUserId")]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserId } )]
+        [Alias("UserId")]
+        [object]
+        $User,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByMembers", ValueFromPipeline = $true)]
+        [int[]]
+        $Member
+    )
+
+    Begin {
+        $id = $UserGroup | Resolve-TeamViewerUserGroupId
+        $resourceUri = "$(Get-TeamViewerApiUri)/usergroups/$id/members"
+        $target = $Member
+        $membersToRemove = @()
+        $null = $ApiToken # https://github.com/PowerShell/PSScriptAnalyzer/issues/1472
+
+        switch ($PsCmdlet.ParameterSetName) {
+            'ByUserGroupMemberId' {
+                $MemberId = $UserGroupMember | Resolve-TeamViewerUserGroupMemberMemberId
+                $resourceUri += "/$MemberId"
+                $target = $UserGroupMember.ToString()
+            }
+            'ByUserId' {
+                $UserId = $User | Resolve-TeamViewerUserId
+                $resourceUri += "/" + $UserId.Substring(1)

```

Comment:
I'd prefer `$User.Id.TrimStart('u')` as we do it, e.g. in `Add-TeamViewerManager`